test/unit-test/cf-atomic-component/utilities/transition/AlphaTransition-spec.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const srcPath = require( '../../src-path' );
```

Comment:
I know this is merged, but `srcPath` is only ever used once in each of these files, do we need to be defining it?

Reply:
NVM, I didn't realize exactly what this was doing till I looked at the other PR that created it.

Reply:
Yeah, just a convenience helper for the test paths.